### DISCUSSION TITLE
Updating the Look

### DISCUSSION THREAD

**Discussion by KhloeLeclair** [eorzea-weather/eorzea-weather#834] [Ideas]
Hello!

I'm glad you liked my pull request for a dark theme. I've been working on some more updates to how the weather table looks and I would like to know what you think. I've added:

* Icons for the different weather types for a bit of visual distinction.
* Summary at top, telling you how long until each type of weather. Can hover estimate for exact time.
* Colored background for the current segment, to make it more obvious.
* One-click option to fade clear/fair skies, to help other weather pop more. Useful for areas with weather-based sprites.
* Options to hide icons and the summary, for people who prefer simpler page.
* The three "Other Settings" persist in the same new settings provider as the dark theme toggle.

Screenshots:
![image](https://user-images.githubusercontent.com/41554123/120431642-c11d9300-c346-11eb-8c86-ce2305a99570.png)
![image](https://user-images.githubusercontent.com/41554123/120432257-aa2b7080-c347-11eb-8079-94dde4aed6ba.png)

I'd also like to include a day/night tracker. Probably just a line in the summary block.

This is still work-in-progress, but it's mostly working. I'm currently debugging performance with React.

Please let me know what you think.

**Comment by KhloeLeclair** [comment]
I've done a bit more work, so here are some fresh screenshots. It hasn't changed significantly, but there is now a day/night timer.

![image](https://user-images.githubusercontent.com/41554123/120601610-2beacf00-c418-11eb-914f-4b412865b7cd.png)
![image](https://user-images.githubusercontent.com/41554123/120601632-3311dd00-c418-11eb-9906-c8cf0ddf962e.png)

As you'll note, it now has a line in the summary showing how long until day or night.

Most of what I've been touching up is behind the scenes. Trying to reduce CPU load and allocations from use of `requestAnimationFrame`. I also ended up updating to a nightly build of the localization library as the stable build is using a positively ancient version of intl-messageformat that doesn't support rich content at all.

I also thought that a summary page for all of Eureka would be nice, and I've come up with this:

![image](https://user-images.githubusercontent.com/41554123/120601930-81bf7700-c418-11eb-94c5-8feb19d8c486.png)
![image](https://user-images.githubusercontent.com/41554123/120601956-85eb9480-c418-11eb-9b0f-11408350a6d2.png)

Clicking a section takes you to the page for that zone. It could also be adopted for Bozja, though it's obviously far less important in Bozja because it only really affects sprite spawning there.

**Comment by flowercartelet** [comment]
Looks good! However, I would like to avoid using the assets that SQEX has the rights to. Is it difficult to find an alternative icon?
